lib/rules/no-incorrect-calls-with-inline-anonymous-functions.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+const utils = require('../utils/utils');
+const ERROR_MESSAGE = 'Do not use anonymous functions as arguments to `scheduleOnce`, `once`, or `debounce`.';
+
+function isAnonymousFunctionExpression(node) {
```

Comment:
Optionally, you could move this function to the `utils/utils.js` file since it's so generic and it's related to the other type checks there.

Reply:
if `utils.isFunctionExpression` is true for named functions, then this method name may not be accurate.  That said a inline named function would also be invalid. Maybe the helper should be renamed to "isAnyFunctionExpression" and moved to utils?